### PR TITLE
Removes mobile-related code & jScrollPane

### PR DESCRIPTION
**Pull request by javierarce**
Removes mobile-related code and, as a bonus, the no-longer used jScrollPane.

Fixes https://github.com/CartoDB/cartodb.js/issues/934.

### CODE REVIEW

**Comment by javierarce** [comment]
cc: @xavijam :pray:

File: grunt/tasks/_browserify-bundles.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 var torqueSpecs = [
   'test/spec/geo/leaflet/leaflet-torque-layer.spec.js',
-  'test/spec/geo/ui/mobile-torque.spec.js',
```
Comment (xavijam) [inline grunt/tasks/_browserify-bundles.js]: But this is mobile torque, should we remove it?

**Comment by xavijam** [comment]
cc @alonsogarciapablo.

**Comment by xavijam** [comment]
What about this code in ```vis.js```:

- https://github.com/CartoDB/cartodb.js/blob/v4/src/vis/vis.js#L328
- https://github.com/CartoDB/cartodb.js/blob/v4/src/vis/vis.js#L701

**Comment by javierarce** [comment]
@xavijam: true, I'll remove that too.

**Comment by xavijam** [comment]
I'd remove that condition, because logo doesn't exist now (as overlay) and share is still to be decided.

**Comment by xavijam** [comment]
And for [this one](https://github.com/CartoDB/cartodb.js/blob/v4/src/vis/vis.js#L485) I'd remove the whole condition too.

File: src/vis/vis.js [hunk 2]
```diff
@@ -482,29 +479,27 @@ var Vis = View.extend({
...
 
-      if (!this.isMobileEnabled) {
-        if (type == 'share' && options['shareable'] || type == 'share' && overlay.model.get('display') && options['shareable'] == undefined) overlay.show();
-        if (type == 'layer_selector' && options[type] || type == 'layer_selector' && overlay.model.get('display') && options[type] == undefined) overlay.show();
-        if (type == 'fullscreen' && options[type] || type == 'fullscreen' && opt.display && options[type] == undefined) overlay.show();
-        if (type == 'search' && options[type] || type == 'search' && opt.display && options[type] == undefined) overlay.show();
-
-        if (type === 'header') {
-          var m = overlay.model;
+      if (type == 'share' && options['shareable'] || type == 'share' && overlay.model.get('display') && options['shareable'] == undefined) overlay.show();
```
Comment (xavijam) [inline src/vis/vis.js]: Shouldn't we avoid same-line conditions?

File: src/vis/vis.js [hunk 3]
```diff
@@ -678,14 +648,6 @@ var Vis = View.extend({
 
     if (opt.force_mobile === false || opt.force_mobile === 'false') this.isMobileEnabled = false;
```
Comment (xavijam) [inline src/vis/vis.js]: IMHO, we should remove ```this.isMobileEnabled``` because we should make any condition with that.

Comment (javierarce) [inline src/vis/vis.js]: Make sense, removing that.

File: src/vis/vis.js (hunk 2 above)
Comment (javierarce) [inline src/vis/vis.js]: Yeah, I'll rewrite them.

**Comment by xavijam** [comment]
Oh!, we forgot to review this, tomorrow! :)

**Comment by xavijam** [comment]
Code looks :+1:, :fire: mobile things!
BTW, I'm removing several images we are not using for new version.

**Comment by javierarce** [comment]
Ok, let me know when I can merge (or merge it yourself)

**Comment by alonsogarciapablo** [comment]
:fire:

**Comment by xavijam** [comment]
Nah, it doesn't make sense, let's do it in a separate PR, go ahead!

**Comment by javierarce** [comment]
:ok_hand:

**Pull request merged**